### PR TITLE
ModMessage: add a codeblock message

### PR DESCRIPTION
**Pull request by IsaacG**

### CODE REVIEW

File: conf.py [hunk 1]
```diff
@@ -55,6 +55,17 @@
         "show off a nice portfolio in the end. Another interesting "
         "resource for web development are the MDN web docs: http://developer.mozilla.org/en-US/docs/Learn."
     ),
+    "codeblock": (
+        "Please use a codeblock when sharing code. It makes the code easier for everyone to read! "
```
Comment (iHiD) [inline conf.py:59]:
```suggestion
        "To make code easier for everyone to read (and to make you look like a pro), **please use a codeblock when sharing code.**"
```

File: conf.py [hunk 2]
```diff
@@ -55,6 +55,17 @@
...
+        "For example, you can type the following:\n\n"
+        "\\`\\`\\`\nfor(int i = 0; i < j; i++) total += total;\n\\`\\`\\`\n\n"
+        "Discord will render that as so:\n"
+        "```\nfor(int i = 0; i < j; i++) total += total;\n```\n\n"
```
Comment (iHiD) [inline conf.py:63]: Screenshot 2023-08-14 at 00 10 56

There's an extra newline somewhere here too.

File: conf.py [hunk 3]
```diff
@@ -55,6 +55,17 @@
...
+        "You can even get syntax highlighting by adding the language to the codeblock opening:\n\n"
+        "\\`\\`\\`cpp\nfor(int i = 0; i < j; i++) total += total;\n\\`\\`\\`\n\n"
```
Comment (iHiD) [inline conf.py:65]:
```suggestion
        "\\`\\`\\`python\nfor(int i = 0; i < j; i++) total += total;\n\\`\\`\\`\n\n"
```

(Most people who don't know how to use codeblocks probably also don't know what `cpp` is as an acronym. I suggest python or JS (and using corrosponding python or JS code)

File: conf.py (hunk 2 above)
Comment (IsaacG) [reply conf.py:63]: Newlines reduced.

File: conf.py (hunk 3 above)
Comment (IsaacG) [reply conf.py:65]: Changed to Python. (I don't know JS.)

**Pull request merged**